src/Media/ReadModel/ImageLDProjector.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Media\ReadModel;
+
+use Broadway\EventHandling\EventListener;
+use CultuurNet\UDB3\EventHandling\DelegateEventHandlingToSpecificMethodTrait;
+use CultuurNet\UDB3\Iri\IriGeneratorInterface;
+use CultuurNet\UDB3\Media\Events\MediaObjectCreated;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+
+final class ImageLDProjector implements EventListener
+{
+    /**
+     * @uses applyMediaObjectCreated
```

Comment:
This seems a bit magical

But is the applyMediaObjectCreated called on a `MediaObjectCreated` event?

And the method is created dynamically here right?
https://github.com/cultuurnet/udb3-backend/blob/master/src/EventHandling/DelegateEventHandlingToSpecificMethodTrait.php#L24